users/resources.py

```diff
@@ -104,6 +104,33 @@ def delete(self, request, pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class StudentPrizesResource(APIView):
```

Comment:
I think that we should keep names consistent - other views are named e.g. `PointResource` and not `StudentPointsResource` (so beside the `student` prefix, please also note single vs plural). For now I'd stick to this convention, we can always adjust later